theme/src/utils/ssg.ts

```diff
@@ -151,6 +152,14 @@ function getGlobalTocHeaders(globalToc: Cheerio<Element>) {
   });
 }
 
+const SEARCH_PAGE_SELECTOR_REMOVE_LIST = [
+  '.footer',
+  '.related',
+  '.logo',
+  '#search-documentation',
+  '.body > p:nth-child(2)',
```

Comment:
why remove the 2nd child?

Reply:
Yeah I suppose it's a bit confusing why I'm removing it each element. I'll add a comment for why each needs to be removed

Reply:
Comments have been added 🔥